jobsets/tests/simple-node-nixops.nix

```diff
@@ -0,0 +1,22 @@
+import ./make-test.nix ({ pkgs, ... }: {
+  name = "boot";
+  nodes = {
+    machine = { config, pkgs, ... }: {
+      imports = [ ../../modules/cardano-node.nix <nixops/nix/options.nix> <nixops/nix/resource.nix> ];
```

Comment:
Shouldn't this use then `cardano-node-aws.nix` if it's testing nixops? Otherwise nixops options aren't used.

But it will barf on `resources` not being populated, hmmm :)